### ISSUE TITLE
Uncaught error when you have a link tag set along with the meta

### ISSUE THREAD

**Issue by traviscrist** [coilhq/web-monetization-projects#850]
If you have both the link tag and 2 meta tags set then this error occurs. It seems like that error should be a caught error. 

Monetization also seems to fail.  

![image](https://user-images.githubusercontent.com/4206970/88566470-e769f800-cfea-11ea-9002-a7a46ecf725b.png)
![Screen Shot 2020-07-27 at 9 08 30 AM](https://user-images.githubusercontent.com/4206970/88566487-ecc74280-cfea-11ea-8fb4-a441d30e6a80.png)

I believe the following cases need to be checked: 
1. meta + link tag should not throw an error
1. multiple meta + link should also not through this error. It's not clear to me what meta is used if there are multiple but I believe that the link should be used if its there. 

@sharafian @sublimator curious on your thoughts here.

**Comment by sublimator** [comment]
My thoughts on this, are put simply, "just don't do that" 

Even before the recent change, multiple meta tags weren't supported, and the error was simply left to be logged in the console. We have only ever allowed one meta tag at a time. 

I can make it work if you want but it can get complicated when you want to handle other cases that seem like they should work while maintaining a 1:1 tag:requestId relationship.

Obviously if we only had to check for the tags once upon page load and weren't tracking them dynamically then it would be a lot simpler.

> meta + link tag should not throw an error

Right, I might have messed that up making it document order dependent! 
I will check.

**Comment by sublimator** [comment]
> Right, I might have messed that up making it document order dependent!

Well yeah, it's order dependent. If there's a link tag added after the meta, then it will stop the meta and start the link. But if it's the other way around, the meta will be ignored. 

Monetization **does** work in both these cases though:
```html
    <link rel="monetization" href="https://twitter.xrptipbot.com/Coil" />
    <meta name="monetization" content="$twitter.xrptipbot.com/Coil" />
```

```html
    <meta name="monetization" content="$twitter.xrptipbot.com/Coil" />
    <link rel="monetization" href="https://twitter.xrptipbot.com/Coil" />
```

**Comment by traviscrist** [comment]
I think we should just be catching the error and throwing a more relevant user message. Instead of the error above it would be great to have it just be an error that you have multiple meta tags so the user becomes aware from the error. I don't think this should be supported. 

It does look like monetization does work so this is really about making the error more user friendly. 

Hmm.. it seems like if there is a link and a meta it should always use the link. @sharafian thoughts?

**Comment by sublimator** [comment]
> Hmm.. it seems like if there is a link and a meta it should always use
the link.
@sharafian <https://github.com/sharafian>thoughts?

With just two tags it should always use the link even now.


Can look into making the error messages nicer.

**Comment by sublimator** [comment]
> so the user becomes aware from the error

I think we'd discussed surfacing various WM configuration errors (SPSP failures etc) before in the UI somehow, but (loosely) decided to just let them be logged in the console ? 

So I've been operating by that.

**Comment by traviscrist** [comment]
I don't think the error should be in the UI. 

We should just provide a relevant console error so when this occurs the console has an appropriate error that tells the user what is happening.

**Comment by sublimator** [comment]
Closed with #930
